frontend/src/pages/Posts.jsx

```
@@ -0,0 +1,326 @@
+      <Navbar />
+      
+      <div className="max-w-7xl mx-auto bg-white rounded-3xl shadow-2xl overflow-hidden mt-16">
+        {/* Header */}
+        <div className="bg-gradient-to-r from-indigo-500 to-purple-600 text-white p-6">
+          <div className="flex justify-between items-center">
+            <h1 className="text-3xl font-bold tracking-wide">Job Posts</h1>
+            {user && user.userType === "Company" && (
+              <button
+                className="bg-white text-purple-600 rounded-full px-6 py-3 shadow-md transition-all hover:bg-gray-100"
+                onClick={() => navigate("/create-post")}
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)